ckanext/depositar_theme/templates/home/index/idx_features.html

```diff
@@ -1,32 +1,17 @@
-<div class="idx idx-features">
+{% set testContent = 'This is a test feature content. This is a test feature content. This is a test feature content. This is a test feature content. This is a test feature content. This is a test feature content.' %}
+{% set featureList = [{'title':'Feature Title 1', 'content':testContent}, {'title':'Feature Title 2', 'content':testContent}, {'title':'Feature Title 3', 'content':testContent}] %}
+
+<div class="idx features">
   <div class="container">
-    <div class="row-bs4 mx-auto">
-      <div class="col">
-        <div>
-          <small class="feature-mark">{{ _('FOR YOUR ––––') }}</small>
-          <h2 class=""> {{ _('Dataset') }}</h2>
-          <p class="">支援眾多檔案格式；易於將資料區分群組、指定標籤，並可針對結構資料視覺化分析，提升被探索及再使用率。</p>
-          <p class="py-3"><a class="btn" href="/">{{ _('more') }}</a></p>
+      {% for feature in featureList %}
+        {% if feature != featureList[0] %}
+            <div class="bar"></div>
```

Comment:
two spaces for indentation

```suggestion
          <div class="bar"></div>
```